Tests/ViewControllers/BrickCollectionViewTests.swift

```diff
@@ -306,7 +314,7 @@ class BrickCollectionViewTests: XCTestCase {
         brickView.layoutSubviews()
 
         var cell = brickView.cellForItemAtIndexPath(NSIndexPath(forItem: 0, inSection: 1)) as? CollectionBrickCell
-        XCTAssertNil(cell?.frame)
+        XCTAssertEqual(cell?.frame.height ?? 0, 0)
```

Comment:
Is the `?? 0` necessary? I though XCTAssertEqual would also compare optional things by default